docs/samples/calling/index.html

```diff
@@ -9,495 +9,515 @@
-        <div id="transfersection">
-          <fieldset>
-            <legend>Call Transfer</legend>
-            <input id="transfer_target" name="transfer_target" value="Transfer Target" type="string">
-            <select name="transfer-options" id="transfer-options">
-              <option>
-                <button id="blind-transfer" type="button" disabled="" class="btn-code">Blind Transfer</button>
-              </option>
-              <option>
-                <button id="consult-transfer" type="button" disabled="" class="btn-code">Consult Transfer</button>
-              </option>
-            </select>
-            <button id="transfer" type="button" onclick="initiateTransfer()" disabled="true"
-              class="btn-code">Transfer</button>
-            <button onclick="endSecondCall()" disabled="" id="end-second" class="btn--red">End Call</button>
-            <pre id="transfer-call">Tranfer call status</pre>
-          </fieldset>
-        </div>
+          </form>
+        </section>
 
-        <!-- calling / incoming -->
-        <div id="incomingsection">
-          <fieldset>
-            <legend>Incoming Call</legend>
+        <!-- calling-management -->
+        <section id="callingManagement">
+          <h2 class="collapsible">Call initialization</h2>
+          <div class="section-content">
+            <fieldset>
+            <legend>Add Local Media</legend>
+            <p class="note">NOTE: In order to send audio/video we must request permissions from the user to access camera
+              and microphone.</p>
             <div class="u-mv">
-              <pre id="incoming-call"> No Incoming Calls</pre>
-              <button onclick="answer()" disabled="" id="answer" class="btn--green">Answer</button>
-              <button onclick="endCall()" disabled="" id="end" class="btn--red">End Call</button>
-              <input onclick="holdResume()" type="button" value="Hold" id="hold_button"></input>
+              <button id="sd-get-media-streams" type="button" onclick="getMediaStreams()"
+                class="btn-code">getMediaStreams()</button>
             </div>
           </fieldset>
-        </div>
-
-        <!--  Call Quality-->
-        <div id="callquality">
-          <fieldset>
-            <legend>Call Quality</legend>
-            <div class="umv">
-              <pre id="call-quality-metrics">Call quality Metrics</pre>
-              <button id="get-call-quality" type="button" onclick="getCallQuality()" class="btn-code">Call
-                Quality</button>
+          
+            <form id="call-create">
+              <fieldset>
+                <legend>Outgoing Call</legend>
+                <div class="u-mv">
+                  <input id="destination" name="destination" placeholder="Destination" type="string" style="display: block; margin-bottom: 0.5rem;">
+                  <button id="create-call-action" type="submit" class="btn-code btn--green">Make Call</button>
```

Comment:
Once we make an outbound call, this button should also get disabled until we disconnect.

Reply:
Done